test/integration/testConnectionPoolExecute.js

```diff
@@ -4,14 +4,16 @@
 
 const async = require('async');
 const testUtil = require('./testUtil');
+const {randomizeName} = require('./testUtil');
 
 describe('Execute test with Pool', function () {
   let connectionPool = null;
-  const createNodeTSQL = 'create or replace table NodeT(colA number, colB varchar)';
-  const selectAllSQL = 'select * from NodeT';
-  const insertNodeTSQL = 'insert into NodeT values(1, \'a\')';
-  const updateNodeTSQL = 'update NodeT set COLA = 2, COLB = \'b\' where COLA = 1';
-  const dropNodeTSQL = 'drop table if exists NodeT';
+  const nodeT= randomizeName('nodeT');
```

Comment:
Do you have schema creation per job? If yes, what is the purpose of these changes?

Reply:
Yes schema is randomised per job but still randomising of table name can protect from parallel running by mocha. It isn't used in ci scripts by every one can use it running scripts.